lib/classes/Error.js

```diff
@@ -2,6 +2,7 @@
 
 const chalk = require('chalk');
 const { inspect } = require('util');
+const isError = require('type/error/is');
```

Comment:
Why don't we use https://lodash.com/docs/4.17.15#isError here?

Reply:
My bad, I didn't actually check if Lodash offers something like that.

I've updated it to use `_.isError` (by looking into internals it looks good, and I agree in context of this single case it's not justified to introduce new dependency).

Still I think at some point it might be good if we derive from lodash, so we rely more on native constructs, and imo `type` stands as nice complement of runtime type validation utils (which is not really covered by lodash) through which we may improve DX experience.